probe-rs/src/debug/mod.rs

```diff
@@ -1823,12 +1997,12 @@ impl<'debuginfo> UnitInfo<'debuginfo> {
                             let pieces = match self.expr_to_piece(core, expression, frame_base) {
                                 Ok(pieces) => pieces,
                                 Err(err) => {
-                                    child_variable.memory_location = u64::MAX;
                                     child_variable.set_value(format!(
                                         "ERROR: expr_to_piece() failed with: {:?}",
                                         err
                                     ));
-                                    return Err(err);
+                                    vec![]
+                                    //return Err(err);
```

Comment:
What's the reason for not returning an error here?

Reply:
Similar to my discussion below. The calling function already (old code) tests for empty vector responses, and this way I can keep the user experience consistent - show the error in context of the variable that caused it, rather than disconnected somewhere in the Console Log.